src/Interfaces/DynamoTestStation.ts

```diff
@@ -15,4 +17,37 @@ interface DynamoTestStation {
   testStationType: string;
```

Comment:
testStationEmail, are we wanting the array here @danielchildsbjss @Ian-Hodges or somewhere else ?

Reply:
I think it would go here, let's hold off until the data analysis has been done though.